<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="4" hostname="Yaswanth" name="com.selenium.dependencies.TestGroupDependency" tests="7" failures="0" timestamp="2022-08-02T21:00:13 IST" time="0.015" errors="1">
  <testcase name="test2" time="0.001" classname="com.selenium.dependencies.TestGroupDependency"/>
  <system-out/>
  <testcase name="test4" time="0.000" classname="com.selenium.dependencies.TestGroupDependency">
    <skipped/>
  </testcase> <!-- test4 -->
  <system-out/>
  <testcase name="test3" time="0.002" classname="com.selenium.dependencies.TestGroupDependency">
    <error type="java.lang.ArithmeticException" message="/ by zero">
      <![CDATA[java.lang.ArithmeticException: / by zero
at com.selenium.dependencies.TestGroupDependency.test3(TestGroupDependency.java:22)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- test3 -->
  <system-out/>
  <testcase name="test1" time="0.012" classname="com.selenium.dependencies.TestGroupDependency"/>
  <system-out/>
  <testcase name="test6" time="0.000" classname="com.selenium.dependencies.TestGroupDependency">
    <skipped/>
  </testcase> <!-- test6 -->
  <system-out/>
  <testcase name="test7" time="0.000" classname="com.selenium.dependencies.TestGroupDependency">
    <skipped/>
  </testcase> <!-- test7 -->
  <system-out/>
  <testcase name="test5" time="0.000" classname="com.selenium.dependencies.TestGroupDependency">
    <skipped/>
  </testcase> <!-- test5 -->
  <system-out/>
</testsuite> <!-- com.selenium.dependencies.TestGroupDependency -->
